cookingrecipe/src/main/java/com/cookingrecipe/dao/GenericDAOImp.java

```diff
@@ -51,4 +51,11 @@ public E getById(Id id) {
 		return res;
 	}
 
+	@Override
+	public E save(E entity) throws Exception {
+		// TODO Auto-generated method stub
+		getHibernateTemplate().saveOrUpdate(entity);
+		getHibernateTemplate().flush();
```

Comment:
cần code line này ah chú

Reply:
ý chị là dòng nào ạ?

Reply:
dòng flush đấy, chị comment dòng đó mà :p

Reply:
em thấy anh Chu hay nhắc là update xong flush, với trong code của anh ấy cũng thế nên em để vậy ạ

Reply:
theo em đọc được thì flush nó sẽ đồng bộ các thay đổi trong session lên db, nhưng chưa commit. Nó được gọi tự động trước khi commit, nên trong commit của transaction cũng đã có flush rồi. Nhưng nếu lượng thay đổi lớn, thì có thể dùng flush để thực hiện thay đổi từng phần nhỏ, sau đó mới commit, sẽ an toàn hơn là để đến cuối cùng mới flush + commit 1 lần ạ.